certbot/tests/compat/misc_test.py

```diff
@@ -1,48 +1,43 @@
 """Tests for certbot.compat.misc"""
 import sys
-import unittest
 from unittest import mock
 
 import pytest
 
 from certbot.compat import os
 
 
-class ExecuteStatusTest(unittest.TestCase):
```

Comment:
I think this class is worth keeping, since there are other utility commands in `certbot.compat.misc` that aren't tested directly by this class